guides/v2.2/magento-functional-testing-framework/release-2/data.md

```diff
@@ -33,15 +33,31 @@ In this example:
 * `name` is a `<data>` key of the entity. The corresponding value will be assigned to `userInput` as a result.
 
 ****
+#### Environmental data
 
 ```xml
-userInput="{{_ENV.MAGENTO_ADMIN_USERNAME}}
+userInput="{{_ENV.MAGENTO_ADMIN_USERNAME}}"
 ```
 
 In this example:
 
 * `_ENV` is a reference to the `dev/tests/acceptance/.env` file, where basic environment variables are set.
-* `MAGENTO_ADMIN_USERNAME` is a name of an environment variable. The corresponding value will be assigned to `userInput` as a result.
+* `MAGENTO_ADMIN_USERNAME` is a name of an environment variable.
+The corresponding value will be assigned to `userInput` as a result.
+
+#### Sensitive data
+
+```xml
+userInput="{{_CREDS.MY_SECRET_TOKEN}}"
+```
+
+In this example:
+
+* `_CREDS` is a reference to the `dev/tests/acceptance/.credentials` file, where sensitive data and secrets are stored for use in a test.
```

Comment:
While it is semi-obvious, perhaps make it explicit that the `.credentials` file is an ".env" style file expecting key value pairs? eg:
```
# This is a .credentials file
MY_SECRET_TOKEN=secrettoken
```